static/js/init.js

```diff
@@ -37,6 +38,16 @@ $('.js-dropdown').each(function() {
   new dropdown.Dropdown(this);
 })
 
+$('.js-typeahead-filter').each(function() {
+    var dataKey = $(this).attr('data-dataset');
+    var datasetMap = {
+        candidates: typeahead.datasets.candidates,
+        committees: typeahead.datasets.committees
+    }
+    var dataset = datasetMap[dataKey];
```

Comment:
Do we need the `datasetMap` variable? How about `var dataset = typeahead.datasets[dataKey];`?